kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/processors/Operator.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+package org.kitodo.mediaserver.core.processors;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Comparison operator representation.
+ */
+public enum Operator {
+    LESS("<"),
+    LESSOREQUAL("<="),
+    GREATER(">"),
+    GREATEROREQUAL(">="),
+    EQUAL("="),
+    NOTEQUAL("!="),
+    CONTAINS(":"),
+    NOTCONTAINS("!:")
```

Comment:
This is really a small matter, but the convention for naming constants in java is with underscore, i.e. GREATER_OR_EQUAL etc., and so it would be easier to read.